software/verification/openmodelica_sim.py

```diff
@@ -0,0 +1,146 @@
+import subprocess
+import os
+import shutil
+
+def run_process(return_dict, cmd, worDir, timeout):
+    """Function to run a subprocess command
+    Parameters:
+    ----------
+    return_dict: dict
+            Contains information about the command
+    cmd: str
+            Command to be run
+    worDir: str
+            Current working directory for the command to run
+    timeout: int
+            Number of seconds before the command times out
+
+    Returns:
+    --------
+    return_dict: dict
+        Contains information about the command
+    """
+
+    output = subprocess.check_output(
+        cmd,
+        # cwd=worDir,
+        # timeout=timeout,
```

Comment:
This is not clear: why is the script taking the working directory and a time out as an argument, but then it is not used? Please correct this. I suggest to keep the time out to make sure there is no deadlock.